Documentation/articles/getting_started/0_welcome.md

```diff
@@ -17,11 +17,3 @@ By the end of this tutorial set, you will have a working project to start buildi
 - [Understanding the Code](2_understanding_the_code.md)
```

Comment:
Need to add a section for the new chapter "Setting up your Development Environment" with links to each page (windows, mac, ubuntu)